### ISSUE TITLE
Sync events auto-generated from Gmail

### ISSUE THREAD

**Issue by phw198** [phw198/OutlookGoogleCalendarSync#1988]
>  My previous question has been to whether people actually _want_ to sync these items....

Yes, some of these items include trips, dentist appointments, etc. that come from gmail emails. I definitely want to sync these items. For example, when I schedule my next dentist appt at the office, my dentist sends me an email confirmation to my gmail. This automatically goes onto my calendar. I use the Gmail -> Outlook sync to get my personal events on my work calendar so that I don't get meetings booked during times where I have personal appts. Right now I am struggling with the latest build because it is excluding these.

_Originally posted by @adamtoth in https://github.com/phw198/OutlookGoogleCalendarSync/issues/1919#issuecomment-2377558923_

As per [comment](https://github.com/phw198/OutlookGoogleCalendarSync/issues/1919#issuecomment-2260870662):
> I'll probably add an option to exclude Gmail events. But if included, for two-way sync I could maybe restrict the attributes that OGCS is trying to update in Google. The [permitted list](https://workspaceupdates.googleblog.com/2024/05/google-calendar-api-event-type-fromgmail.html) is reminders, colorId, visibility, status, and extendedProperties.

👀 **Manual Workaround:** To sync a Google auto-created item, in https://calendar.google.com click the item's kebab menu icon and select `Duplicate` - OGCS will sync the duplicate. Optionally, delete the original item.

**Comment by doczong** [comment]
Add me to those that would like those sync'd.  I swear they sync'd before but maybe not.  All my flights, trains, hotels and dinner reservations show up in google but not in outlook.  Hourly appts and full day (like hotels).

**Comment by phw198** [comment]
@doczong Noted, but unfortunately, v2 of OGCS is mostly in bug-fix support now and focus is on v3 which will no longer require the Outlook client. Until v3 is out, only very simple features will make the cut for v2.

**Comment by shotgunheist** [comment]
Adding my support for this. Is there a way to respond to these in gmail so they will show up in outlook? Currently using OGCS for gmail > outlook sync. "Accepting" these auto generated events doesn't seem to push to outlook under any settings. Maybe an automatic duplication of the event to a "normal" event on g cal would make it show up in outlook?

**Comment by shotgunheist** [comment]
Noting that I have a solution for now: i'm using a different program that automatically duplicates these events on my gcal when they show up, which then will be picked up by OGCS!

**Comment by CKMendo** [comment]
> Noting that I have a solution for now: i'm using a different program that automatically duplicates these events on my gcal when they show up, which then will be picked up by OGCS!

What program are you using to duplicate these items?   I'm losing track of everything between google Cal and Outlook since they are not syncing and need a solution...  thanks!

**Comment by phw198** [comment]
@CKMendo If you go to the the Google created item, click the kebab menu icon and select `Duplicate`, does the duplicated item get synced by OGCS?

**Comment by CKMendo** [comment]
Yes, if I duplicate the item as you suggested, it syncs properly.
  

  

  

  
  
  
  
  
>

**Comment by openr0ad** [comment]
I'd also like to throw my support behind this enhancement. Duplicating and deleting is a tedious workaround

**Comment by openr0ad** [comment]
I used Claude to help me build a Google Apps Script workaround for this while waiting for a proper fix — duplicates fromGmail events so OGCS can sync them. Repo here: https://github.com/openr0ad/gmail-event-ogcs-sync
